library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -151,8 +152,17 @@ void PlatformBridgeFilter::onDestroy() {
   platform_filter_.instance_context = nullptr;
 }
 
+void PlatformBridgeFilter::FilterBase::dumpState(std::ostream&, int) const {
+  // TODO(junr03): output to ostream - https://github.com/envoyproxy/envoy-mobile/issues/1497.
+  ENVOY_LOG(error, "PlatformBridgeFilter: {} state={} stream_complete={} error_response={}",
```

Comment:
@goaway anything else you would want to see related to the PBF state?

Reply:
Status of the buffer (does it exist? is anything buffered? how much?) would be great.

Reply:
Also whether headers/data/trailers have been forwarded.

Reply:
sg. Will unearth.

Reply:
@goaway this is updated!